src/reboost/hpge/psd.py

```diff
@@ -460,36 +764,60 @@ def maximum_current(
         Array of energies for each step.
     drift_time
         Array of drift times for each step.
-    sigma
-        Sigma parameter of the current pulse model.
-    tail_fraction
-        Tail-fraction parameter of the current pulse.
-    tau
-        Tail parameter of the current pulse
-    mean_AoE
-        The mean AoE value for this detector (to normalise current pulses).
-    get_timepoint
-        Flag to return the time of the maximum current (relative to t0) instead of the current.
+    dist_to_nplus
+        Distance to n-plus electrode, only needed if surface heuristics are enabled.
+    template
+        array of the bulk pulse template
+    times
+        time-stamps for the bulk pulse template
+    fccd
+        Value of the full-charge-collection depth, if `None` no surface corrections are performed.
+    surface_library
+        2D array (distance, time) of the rate of charge arriving at the p-n junction. Each row
+        should be an array of length 10000 giving the charge arriving at the p-n junction for each timestep
+        (in ns). This is produced by `reboost.hpge.surface.get_surface_response` or other libraries.
```

Comment:
```suggestion
        (in ns). This is produced by :func:`.hpge.surface.get_surface_response` or other libraries.
```
I don't remember if this works